plugin/pkg/auth/authenticator/token/webhook/webhook.go

```diff
@@ -64,11 +65,15 @@ func (w *WebhookTokenAuthenticator) AuthenticateToken(token string) (user.Info,
 		if err := result.Error(); err != nil {
 			return nil, false, err
 		}
+		var statusCode int
+		if result.StatusCode(&statusCode); statusCode < 200 || statusCode >= 300 {
```

Comment:
We don't want to cache error responses from the webhook server at all. We do want to cache our successful "No" responses.